backends/advanced/src/advanced_omi_backend/processors.py

```
@@ -445,12 +458,268 @@ def cleanup_processing_tasks(self, client_id: str):
+        """Calculate pipeline performance metrics for each processing stage."""
+        import time
+        from statistics import mean
+
+        current_time = time.time()
+
+        # Calculate stats for each queue
+        pipeline_stats = {}
+
+        # Audio Queue Stats
+        audio_tasks = []
+        for client_id, state in self.processing_state.items():
```

Ruff (0.13.1)

Loop control variable `client_id` not used within loop body

Rename unused `client_id` to `_client_id`

(B007)